components/VideoPlayer.vue

```diff
@@ -16,139 +16,138 @@
      */
     playsinline: {
       type: Boolean,
-      default: false
+      default: false,
+    },
+  },
+  data() {
+    return {
+      showSound: false,
+      mouseover: false,
+      playing: false,
+      started: false,
+      loading: false,
+      time: 0,
+      duration: 0,
+      volumeInternal: 1,
+      fullscreenInternal: false,
     }
   },
   computed: {
     /**
      * @private
      */
     srcComputed() {
-      if (typeof this.src == "string") return this.src;
-      return null;
+      if (typeof this.src === 'string') return this.src
+      return null
     },
-   
+
     /**
      * @private
      */
     playingComputed: {
       get() {
-        if (this.value != undefined) return this.value;
+        if (this.value !== undefined) return this.value
 
-        return this.playing;
+        return this.playing
       },
 
       set(e) {
-        if (this.value != undefined) {
-          this.$emit("input", e);
+        if (this.value !== undefined) {
+          this.$emit('input', e)
         } else {
-          this.playing = e;
+          this.playing = e
         }
-      }
-    }
-  },
-  data() {
-    return {
-      showSound: false,
-      mouseover: false,
-      playing: false,
-      started: false,
-      loading: false,
-      time: 0,
-      duration: 0,
-      volumeInternal: 1,
-      fullscreenInternal: false
-    };
+      },
+    },
   },
   watch: {
     value(after) {
       if (after && this.$refs.video.paused) {
-        this.$refs.video.play();
+        this.$refs.video.play()
       } else if (!after && !this.$refs.video.paused) {
-        this.$refs.video.pause();
+        this.$refs.video.pause()
       }
     },
 
     volume() {
-      this.$refs.video.volume = this.volumeComputed;
-    }
+      this.$refs.video.volume = this.volumeComputed
+    },
   },
   methods: {
     /**
      * @private
      */
     atEnded() {
-      console.log("video ended");
+      console.log('video ended')
     },
     /**
      * @private
      */
     atVolumechange() {
-      this.volume = this.$refs.video.volume;
-      console.log("volume changed");
+      this.volume = this.$refs.video.volume
+      console.log('volume changed')
     },
     /**
      * @private
      */
     atTimeupdate() {
-      this.time = this.$refs.video.currentTime;
-      console.log("time updated", this.time);
+      this.time = this.$refs.video.currentTime
+      console.log('time updated', this.time)
     },
     /**
      * @private
      */
     atPlayPause() {
-      if (!this.playingComputed) this.$refs.video.play();
-      else this.$refs.video.pause();
+      if (!this.playingComputed) this.$refs.video.play()
+      else this.$refs.video.pause()
     },
     /**
      * @private
      */
     play() {
-      this.playingComputed = true;
-      this.started = true;
-      console.log("Called play");
+      this.playingComputed = true
+      this.started = true
+      console.log('Called play')
```

Comment:
do you wanna keep the console.logs?

Reply:
This was actually not my code, @pushpanjalip what is your opinion on these logs?

Reply:
I have added console log to just show that specific event is being captured